articles/extensions/authorization-extension/v2/implementation/setup.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Setting Up the Authorization Extension
+description: How to set up the Authorization Extension
+toc: true
+---
+
+# Authorization Extension: Setup
+
+Once you've configured the extension, you can manage your users, as well as the groups, roles and permissions for your users in the Authorization Extension dashboard.
```

Comment:
The first sentence and the Entities Available header after it, feel redundant. Start the doc with the `In this paragraph, we'll briefly cover the basics of users, groups, roles, and permissions....` paragraph, it looks good 👍